packages/simple-hub/src/wallet/xstate-wallet-internals/index.ts

```diff
@@ -309,3 +317,13 @@ export const firstState = (
   participants,
   outcome
 });
+
+export function makeDestination(addressOrDestination: string): string {
```

Comment:
Should [`deposit.ts`](https://github.com/statechannels/monorepo/blob/ags/wallet-fixes/packages/simple-hub/src/wallet/deposit.ts#L35) be updated to use this function?